src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/WindowsServiceLifetime.cs

```diff
@@ -109,6 +135,10 @@ protected override void OnShutdown()
             base.OnShutdown();
         }
 
+        /// <summary>
+        /// Releases the resources used by the <see cref="WindowsServiceLifetime"/>.
+        /// </summary>
+        /// <param name="disposing"><c>true</c> is invoked from <see cref="IDisposable.Dispose"/>.</param>
```

Comment:
```suggestion
        /// <param name="disposing">
               <see langword="true /> only when called from <see cref="IDisposable.Dispose"/>; otherwise, <see langword="false" />.</param>
```